go/vt/vtgate/planbuilder/symtab.go

```diff
@@ -400,10 +400,23 @@ func (st *symtab) ResolveSymbols(node sqlparser.SQLNode) error {
 // It represents a table alias in a FROM clause. It points
 // to the builder that represents it.
 type table struct {
-	alias       sqlparser.TableName
-	columns     map[string]*column
-	origin      builder
-	vindexTable *vindexes.Table
+	alias         sqlparser.TableName
+	columns       map[string]*column
+	orderdColumns []sqlparser.ColIdent
```

Comment:
typo -- orderedColumns

Reply:
Also FWIW I think calling this `columnNames` is sufficient. It's self-evidently ordered from the fact that it's an array.